src/content/core/remote-execution.mdx

```diff
@@ -12,7 +12,9 @@ By default, your commands run locally in your machine, but we recommend that you
 
 ## Enable remote execution
 
-Enable remote execution by setting the `remote` field to `true` in the deploy/destroy section of your Okteto Manifest:
+You can enable remote execution by default for all applications on your Okteto cluster by [following these instructions](#enable-remote-execution-by-default).
+
+To use remove execution per application, you can enable it by setting the `remote` field to `true` in the deploy/destroy section of your Okteto Manifest:
```

Comment:
```suggestion
To use remote execution per application, you can enable it by setting the `remote` field to `true` in the deploy/destroy section of your Okteto Manifest:
```

Reply:
We already have this paragraph to document this:
https://deploy-preview-935--okteto-docs.netlify.app/docs/1.27/core/remote-execution/#enable-remote-execution-by-default
We should refactor this info to have a single place that explains how to enable remote execution by default

Reply:
Bringing that section up to consolidate